RecoLocalMuon/RPCRecHit/plugins/DTSegtoRPC.cc

```diff
@@ -161,6 +123,19 @@ std::unique_ptr<RPCRecHitCollection> DTSegtoRPC::thePoints(const DTRecSegment4DC
         float dy = segmentDirection.y();
         float dz = segmentDirection.z();
 
+        float myPhiTime = -9999.;
+        float myPhiTimeErr = -9999.;
+        int myPhiBx = -99;
```

Comment:
these placeholder magic number are repeating several times. can you define them in a single place?